### ISSUE TITLE
Setup instructions Slack channel link gives 404

### ISSUE THREAD

**Issue by nmb-paperspace** [gradient-ai/Graphcore-Tensorflow2#20]
"Running on Paperspace" cell links to https://nbwbgsfvwu.clehbtvty.paperspacegradient.com/files/learning-TensorFlow2-on-IPU/mnist/graphcorecommunity.slack.com?token=358...81b (token abbreviated), which gives a Jupyter 404.

I assume this is the same on all notebooks since the instructions are merged as part of the repo setup.

**Comment by nmb-paperspace** [comment]
Swapping this to Rahul since it's the same issue as with PyTorch
